ow_plexil/src/plans/TestActions.plp

```diff
@@ -30,9 +30,7 @@ TestActions:
   LibraryCall DigLinear (X = 1.75, Y = 0.1, Depth = 0.045, Length = 0.1,
                          GroundPos = GP);
 
```

Comment:
[optional] Since this plan is intended to test all actions, a call to Discard could be placed at line 25, right after the DigCircular.

Reply:
Good observation.  Added!